app/models/publisher.rb

```diff
@@ -255,6 +268,12 @@ def uphold_status
       :access_parameters_acquired
     elsif self.uphold_code.present?
       :code_acquired
+    elsif self&.wallet&.uphold_account_status == UpholdAccountState::RESTRICTED || (self.wallet.present? && self.uphold_verified? && self.wallet.not_a_member?)
+      UpholdAccountState::RESTRICTED
```

Comment:
I'm not getting into this state because the incomplete is returning true first. I don't think we need the incomplete if we're going to be displaying this during restricted status